doozerlib/operator_metadata.py

```diff
@@ -584,46 +584,40 @@ class OperatorMetadataLatestNvrReporter:
     def __init__(self, operator_nvr, stream, runtime):
         self.operator_nvr = operator_nvr
         self.stream = stream
+        self.runtime = runtime
 
-        self.operator_name, self.operator_version, self.operator_release = self.unpack_nvr(operator_nvr)
+        self.operator_component, self.operator_version, self.operator_release = self.unpack_nvr(operator_nvr)
 
-        self.metadata_name = '{}-metadata'.format(self.operator_name)
+        self.metadata_component = self.operator_component.replace('operator-container', 'operator-metadata-container')
```

Comment:
Replacing `-container` with `-metadata-container` seems to be more robust without considering whether the operator name is ending with `-operator`  (is that possible?)

Reply:
seemed conceivable to me that some operator would have `-container` in its name.
`my-super-container-manager-operator-container`
this could probably use an assertion that there was actually something to replace, though.